gnoboard/src/hooks/use-gno.ts

```diff
@@ -10,32 +15,70 @@ interface GnoResponse {
   setPassword: (password: string) => Promise<void>;
 }
 
+// TODO: This is a hack to avoid creating multiple GRPC clients. We should
+// create a single client and pass it around.
+let clientInstance = undefined;
+let bridgeInstance: boolean = false;
+
 export const useGno = (): GnoResponse => {
-  const createAccount = async (nameOrBech32: string, mnemonic: string, password: string) => {
-    console.log('nameOrBech32', nameOrBech32);
-    console.log('mnemonic', mnemonic);
+  const getClient = async () => {
+    if (!bridgeInstance) {
+      console.log('Initializing bridge...');
+      await GoBridge.initBridge();
+      bridgeInstance = true;
+    }
+
+    if (clientInstance) return clientInstance;
+
+    console.log('Creating GRPC client instance...');
 
-    return await GoBridge.createAccount(nameOrBech32, mnemonic, '', password, 0, 0);
+    const port = await GoBridge.getTcpPort();
+    clientInstance = await Grpc.createClient(port);
+    return clientInstance;
+  };
+
+  const createAccount = async (nameOrBech32: string, mnemonic: string, password: string) => {
+    const client = await getClient();
+    const reponse = await client.createAccount(
+      new CreateAccountRequest({
+        nameOrBech32,
+        mnemonic,
+        password,
+      }),
+    );
+    return reponse.key;
   };
 
   const generateRecoveryPhrase = async () => {
-    return await GoBridge.generateRecoveryPhrase();
+    const client = await getClient();
+    const response = await client.generateRecoveryPhrase();
+    return response.Phrase;
   };
 
   const initBridge = async () => {
```

Comment:
`initClient` already calls `GoBridge.initBridge()`. I think we should remove this redundant function.